README.markdown

```diff
@@ -216,6 +217,19 @@ nvm install 6 --reinstall-packages-from=5
 nvm install v4.2 --reinstall-packages-from=iojs
 ```
 
+### Default global packages from file while installing
+
+If you have a list of default packages you want installed every time you install a new version we support that too. You can add anything npm would accept as a package argument on the command line.
+
+```sh
+# $NVM_DIR/default-packages
+
+gulp
+webpack
+bower@1.7.1
```

Comment:
let's avoid using bower as an example here too (i can update this myself prior to merging tho, not a blocker)